src/intl/en/learn-quizzes.json

```
@@ -324,5 +324,52 @@
+  "i004-prompt": "What hard drive storage is required for an Ethereum node?",
+  "i004-a-label": "512 GB SSD",
+  "i004-a-explanation": "Currently, no client software is capable of storing the chain using only 512 GB",
+  "i004-b-label": "2 TB Rotating",
+  "i004-b-explanation": "Generally speaking, rotating hard disks do not support the read/write speeds needed to keep up with processing requirements for an Ethereum node, and an SSD drive is recommended",
+  "i004-c-label": "2 TB SSD",
+  "i004-c-explanation": "At time of writing, a 2 TB SSD drive should satisfy the storage and read/write speed requirements for a full Ethereum node.",
+  "i004-d-label": "8 TB SSD",
+  "i004-d-explanation": "At time of writing, a 2 TB SSD drive should satisfy the storage and read/write speed requirements for a full Ethereum node. An 8 TB SSD would allow for more future-proofing, and the ability to also sync layer 2 chains, but is not currently a requirement for Mainnet.",
+  "i005-prompt": "What happens if your node goes offline?",
+  "i005-a-label": "Your node drops out of sync with the current state of the network",
+  "i005-a-explanation": "When your node is not available online, it is unable to receive new transactions and blocks from peers, and as such falls out-of-sync with the current state of the chain. Connecting back online will allow your node software to get synced back up to be fully functional again.",
+  "i005-b-label": "The ETH in your cold storage is slashed",
+  "i005-b-explanation": "ETH held in your cold storage has nothing to do with your node being online or not. If your node is offline, you won't be able to use it to look up the latest balance of your accounts, but being offline does not put your secured funds at risk. If you're also running validator software with your node as a staker, small penalties will be incurred to this validator balance while unavailable to the network.",
+  "i005-c-label": "The energy used looking for proof-of-work is wasted",
+  "i005-c-explanation": "Ethereum no longer uses proof-of-work, and this was never a requirement of all node operators. Being offline simply means your node is no longer in sync with the latest changes on the network, and can re-sync by returning online.",
+  "i005-d-label": "Chain data is removed, and re-syncing from scratch is required",
+  "i005-d-explanation": "Simply going offline does not typically delete any saved chain data. Connecting back to the internet will allow the software to resume where it left off to sync up with the latest transactions.",
+  "i006-prompt": "Running a node earns network rewards",
+  "i006-a-label": "True",
+  "i006-a-explanation": "Simply running client software to hold and verify your own copy of Ethereum does not earn you rewards. With a node set up, users can chose to also stake their ETH from home to participate in consensus and earn network rewards—this is not required to run a node, nor automatic.",
```

a bit superfluous to the question 

```suggestion
  "i006-a-explanation": "Simply running client software does not earn you rewards. To earn rewards, you must also be staking.",
```